benchmark-framework/src/main/java/io/openmessaging/benchmark/WorkloadGenerator.java

```diff
@@ -76,10 +76,11 @@ public TestResult run() throws Exception {
         List<String> topics = worker.createTopics(new TopicsInfo(workload.topics, workload.partitionsPerTopic));
         log.info("Created {} topics in {} ms", topics.size(), timer.elapsedMillis());
 
-        createConsumers(topics);
         createProducers(topics);
+        createConsumers(topics);
```

Comment:
The sequence of creating consumers before producer was deliberate, since that will trigger the creation of subscriptions ensuring the consumer will not miss the initial data. 

What is the reason for this change?

Reply:
Is this deliberate for other drivers? I'm going to review my branch to see if it matters.

Regarding initial data isn't the benchmark warming up and then resetting stats? Does the initial data matter and in what scenario?

Reply:
Reverting that change is looking good, but it's increased my publish latency mean and median averages.

I'll add a new commit tomorrow.